### ISSUE TITLE
PHP 7.1.5

### ISSUE THREAD

**Issue by Ewwe** [Lapayo/SAMPHP#13]
Hi i will be really glad if you build plugin with php 7.1.5 or tell me how i can build it with this version. Really thank you for response

**Comment by AmirSavand** [comment]
Yes, on linux, it's already the latest version but not on windows.

**Comment by Ewwe** [comment]
And will be on win soon ?
